python/cuml/dask/common/dask_df_utils.py

```diff
@@ -79,3 +81,29 @@ def to_dask_cudf(futures):
     dfs = [d for d in futures if d.type != type(None)]  # NOQA
     meta = c.submit(get_meta, dfs[0]).result()
     return dd.from_delayed(dfs, meta=meta)
+
+
+def to_dask_df(dask_cudf):
+    """
+    Convert a Dask-cuDF into a Pandas-backed Dask Dataframe.
+
+    :param dask_cudf : dask_cudf.DataFrame
+    :return : dask.DataFrame
+    """
+
+    def to_pandas(df, r):
+        return df.to_pandas()
+
+    c = default_client()
+    delayed_ddf = dask_cudf.to_delayed()
+    gpu_futures = c.compute(delayed_ddf)
+    wait(gpu_futures)
```

Comment:
do we need to wait for these here? could we just pass in the futures and let it rip? I thought that should work.

Reply:
This `wait` is only waiting for new futures to be returned from the workers. It's not actually waiting on a computation. While i think this is useful in asynchronous functions to `yield` so that other threads can time in the pool, I do think this can be safely removed here.